server/app/routes/carts/index.js

```diff
@@ -46,6 +51,58 @@ router.delete('/:id', function(req, res, next){
 })
 
 
+// Add item to items list
+router.post('/:id', function(req, res, next){
+    Cart.findById(req.params.id)
+        .then(function(cart){
+            var index =  _.find(cart.items, { 'product': req.body.product });
+            if (index != -1) {
+                cart.items[index].quantity += req.body.quantity;
```

Comment:
Depending on the UI, I think we might want to do: 
cart.items[index].quantity = req.body.quantity;

If the customer specifies they want 2 of the item, we don't want to add 2 to the existing one item that they have in the cart?